lib/evss/disability_compensation_form/service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'common/client/concerns/monitoring'
+
+module EVSS
+  module DisabilityCompensationForm #526
+    class Service < EVSS::Service
+      include Common::Client::Monitoring
+
+      configuration EVSS::DisabilityCompensationForm::Configuration
+
+      def get_rated_disabilities
+        with_monitoring do
+          raw_response = perform(:get, 'ratedDisabilities')
+          RatedDisabilitiesResponse.new(raw_response.status, raw_response)
+        end
+      rescue StandardError => e
+        handle_error(e)
+      end
+
+      def submit_form(form_content)
+        with_monitoring do	
+          headers = { 'Content-Type' => 'application/json' }
+          raw_response = perform(:post, 'submit', form_content, headers)
+          FormSubmitResponse.new(raw_response.status, raw_response)
+        end
+      end
+
+      private
+
+      def handle_error(error)
+        if error.is_a?(Common::Client::Errors::ClientError) && error.status != 403 && error.body.is_a?(Hash)
+          log_message_to_sentry(
+            error.message, :error, extra_context: { url: config.base_path, body: error.body }
+          )
```

Comment:
What error condition is this handling?  @kreek , can you speak to [what it handles in `letters`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/letters/service.rb#L42-L45)?

Reply:
@egarreau the first thing to do on this is find out what the possible error keys are (would be helpful if it was in EVSS's swagger but it's not). On the off chance that they don't have any for 526 you wouldn't need an error mapping class and could just raise a generic backend service exception.